apps/bixarena/app/bixarena_app/main.py

```diff
@@ -318,8 +319,27 @@ def build_app():
             outputs=pages + [leaderboard_metrics],
         )
         cta_btn.click(
-            lambda: navigator.show_page(1) + example_prompt_ui.refresh_prompts(),
+            lambda: handle_start_evaluation_click(
+                navigator, example_prompt_ui.refresh_prompts
+            ),
             outputs=pages + prompt_outputs,
+            js="""
+() => {
+  // Check if user is authenticated by checking if login button says "Logout"
```

Comment:
The proper way to check if the user is authenticated is to use function `user_state.is_authenticated()`. The home page should already have all the logic needed to show the user stats when the user returns and is logged in.